### ISSUE TITLE
Documentation contains deprecated code

### ISSUE THREAD

**Issue by ValueRaider** [requests-cache/requests-cache#732]
https://requests-cache.readthedocs.io/en/stable/user_guide/inspection.html

> You can use CachedSession.cache.urls to see all URLs currently in the cache:
``` Python
session = CachedSession()
print(session.cache.urls)
```

Problem is now, that code produces this deprecation warning:
> requests_cache/backends/base.py:291: DeprecationWarning: BaseCache.values() is deprecated; please use .filter() instead

Also documentation doesn't cover `filter()`, not for a newbie. I just want to see if a url in cache.

**Comment by JWCook** [comment]
Thanks for pointing that out. I'll fix that soon.

>  I just want to see if a url in cache.

You can use the `contains()` method for that. Example:
```python
from requests_cache import CachedSession

session = CachedSession()
session.cache.contains(url='https://example.com')
```

**Comment by JWCook** [comment]
That `DeprecationWarning` has been fixed in 0.9.8.

Some more examples using `filter()` have been added to the docs for the next release. Here is a preview (pre-release docs): https://requests-cache.readthedocs.io/en/main/user_guide/inspection.html
